README.md

```diff
@@ -73,40 +73,90 @@ A filtering function for modifying a tile's features and properties to support l
-    - If a feature has a worldview value of `all` it is considered a match and `worldview: all` is added to the feature's properties and the `params.worldview_property` property is dropped. If the original feature contains a `worldview` property, it is ovewritten.
-    - If a feature does not match the request worldview the entire feature is dropped.
-    - If a feature does not have a `params.worldview_property` property it is retained.
-  - `params.worldview_property` **String** the name of the property that specifies in which worldview a feature belongs. The vector tile encoded property must contain a comma-separated string of ISO 3166-1 alpha-2 country codes that define which worldviews the feature represents (example: `US,RU,IN`). Default value: `_mbx_worldview`.
-    - If a feature contains multiple values that match multiple given values in the `params.worldview` array, it will be split into multiple features in the final vector tile, one for each matching worldview.
-  - `params.class_property` **String** the name of the property that specifies the class of a feature (examples: `sea`, `canal`, `village`). Default value: `_mbx_class`.
+  - `params.buffer` **Buffer** a vector tile buffer, gzip compressed or not.
+  - `params.compress` **Boolean** a boolean value indicating whether or not to return a compressed buffer.
+    - Default value: `false` (i.e. return an uncompressed buffer).
+  - `params.languages` **Array<String>** array of IETF BCP 47 language codes used to search for matching translations available in a feature's properties.
+    - Optional parameter.
+    - All language-related properties must match the following format: `{language_prefix}{language_property}_{language}`.
+    - Default properties are `_mbx_name_{language}`; for example, the `_mbx_name_jp` property contains the Japanese translation for the value in `name`.
+  - `params.language_property` **String** the primary property in features that identifies the feature in a language.
+    - Default value: `name`.
+    - This values is used to search for additional translations that match the following format `{language_property}_{language}`.
+  - `params.language_prefix` **String** prefix for any additional translation properties.
+    - Default value: `_mbx_`.
+    - The value is used to search for additional translations that match the following format: `{language_prefix}{language_property}_{language}`.
+  - `params.worldviews` **Array<String>** array of ISO 3166-1 alpha-2 country codes used to filter out features of different worldviews.
+    - Optional parameter.
+    - For now, only the first item in the array will be processed; the rest are discarded. (*TODO in the future*: expand support for more than one worldviews.)
+  -`params.worldview_property` **String** the name of the property that specifies in which worldview a feature belongs.
```

Comment:
maybe need a newline above? this isn't rendering on its own line in github for me

Reply:
Missing a space before `params.worldview_property`. Fixed in 3711e05918f95ff3494302a0c36e07fe1bdfad08.